src/input/mpegts/satip/satip.c

```diff
@@ -384,12 +398,6 @@ satip_device_hack( satip_device_t *sd )
     sd->sd_pids_max    = 128;
     sd->sd_pids_len    = 2048;
     sd->sd_no_univ_lnb = 1;
-  } else if (strstr(sd->sd_info.manufacturer, "AVM Berlin") &&
```

Comment:
Think about this in other way - user will override the default settings and it won't work with the current firmware. Again, if you like, add option to disable all known workarounds, so users can hack the settings without any auto-things.. But... keep these auto things in one place (hack function).